packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

```diff
@@ -8,61 +8,41 @@ As a developer building a custom application, you want to give your users the fr
 
 What is an Identity Provider? It is a service that creates and maintains identity information and then provides authentication services to your applications. Identity Providers can significantly reduce sign-in and registration friction, which allows your users to easily access applications without needing to create new passwords or remember usernames.
 
-Currently Okta supports the following social Identity Providers:
-
-- [Amazon](/docs/guides/social-login/amazon/main)
-- [Apple](/docs/guides/add-an-external-idp/apple/main/)
-- [Azure](/docs/guides/add-an-external-idp/azure/main/)
-- [Facebook](/docs/guides/add-an-external-idp/facebook/main/)
-- [GitHub](/docs/guides/social-login/github/main)
-- [GitLab](/docs/guides/social-login/gitlab/main)
-- [Google](/docs/guides/add-an-external-idp/google/main/)
-- [LinkedIn](/docs/guides/add-an-external-idp/linkedin/main/)
-- [Microsoft](/docs/guides/add-an-external-idp/microsoft/main/)
-- [PayPal](/docs/guides/social-login/paypal/main)
-- [Salesforce](/docs/guides/social-login/salesforce/main)
-- [Yahoo](/docs/guides/social-login/yahoo/main)
-- [Yahoo Japan](/docs/guides/social-login/yahoojp/main)
-
-Additionally, Okta supports the following Identity Providers:
-
-- [Okta to Okta](/docs/guides/add-an-external-idp/oktatookta/main/)
-- [OpenID Connect](/docs/guides/add-an-external-idp/openidconnect/main/)
-- [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/)
+[View and configure Okta's Identity Provider integrations](/docs/guides/identity-providers/)
```

Comment:
Any way to make this pop out a bit? Button?

Reply:
We can do that - but I believe that @albertchen-okta's team would be the people to ask for that.